Autopilot/AttitudeManager/InputCapture.h

```diff
@@ -19,10 +19,11 @@
 #define USE_PPM 1
 
 /**
- * If the PPM signal is inverted. Some receivers, such as the ezUHF have an inverted
- * PPM signal, so set this to 1. If using the OrangeRX, set this to 0
+ * If the PPM signal is inverted (inverted = active low). Some receivers, such 
+ * as the OrangeRX have an inverted PPM signal, so set this to 1. If using the 
+ * ezUHF, set this to 0
  */
-#define PPM_INVERTED 0
+#define PPM_INVERTED 1 // Might not be necessary? Single edge mode will catch either pulse
```

Comment:
I dont like having unactionable comments like this in the code base. Delete the comment and create a github issue or an asana task to test whether this flag does anything by testing with the ezuhf. You dont have to do it but create it so that its tracked